shell/src/test/java/tachyon/shell/TFsShellTest.java

```diff
@@ -492,15 +540,15 @@ public void renameTest() throws IOException {
     Assert.assertNull(mTfs.getFile(new TachyonURI("/testFolder1")));
   }
 
-  @Test
+  @Test(expected = IOException.class)
   public void renameToExistingFileTest() throws IOException {
     StringBuilder toCompare = new StringBuilder();
     mFsShell.run(new String[] {"mkdir", "/testFolder"});
     toCompare.append(getCommandOutput(new String[] {"mkdir", "/testFolder"}));
     mFsShell.run(new String[] {"mkdir", "/testFolder1"});
     toCompare.append(getCommandOutput(new String[] {"mkdir", "/testFolder1"}));
-    Assert
-        .assertEquals(-1, mFsShell.rename(new String[] {"rename", "/testFolder1", "/testFolder"}));
```

Comment:
Now rename has no return value, so all we can really assert is that the IOException is thrown.